tools/webpack.config.dev.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+var version = require('./../package.json').version;
+var path = require('path');
+var webpack = require('webpack');
+var autoprefixer = require('autoprefixer');
+
+var rootPath = path.join(__dirname, '..');
+
+function root(p){
+  return path.join(rootPath, p);
+}
+
+var distPath = root('dist');
+
+module.exports = {
+  entry: [
+    'webpack-hot-middleware/client?reload=true',
+    './src/entry'
+  ],
+  output: {
+    path: distPath,
+    publicPath: '/dist',
+    filename: 'bundle.js'
```

Comment:
Yep, there definitely will be a prod config.
But the prod process in our case is a different big story. I've started with the components migration first.

I am not sure about chunks, though. It need deep consideration for our style guide usage patterns.

`bundle.js` will exist, presumable, only in development!